frontend/src/metabase/query_builder/components/NativeQueryEditor/AceEditor/AceEditor.tsx

```diff
@@ -0,0 +1,570 @@
+import type { Ace } from "ace-builds";
+import * as ace from "ace-builds/src-noconflict/ace";
+import {
```

Comment:
:x: Collapsing blocks does not work

https://github.com/user-attachments/assets/d6bfa82d-c2bb-46fb-8dd4-0f885a127594

Reply:
BTW in this video the following parts have wrong colors: `COUNT`, `MIN`, `889`, `7`, `1048575`

Reply:
I cannot reproduce the failing collapsing blocks on all three browsers. Does this work for you on master?

Reply:
I can reproduce it in Firefox and Chrome.
It works on `master` in both browsers.

I see it depends on where you click:
- clicking precisely in the middle of the button does not work
- clicking on the right edge of the button works

https://github.com/user-attachments/assets/c424bf51-913b-4d43-9817-761561314322

Reply:
I've pushed a change that reduces the impact of this a little bit. I don't want to spend too much time on this since we're getting rid of the ACE editor anyways.

AFAICT this also happens on stats:

https://github.com/user-attachments/assets/d4ee2776-ddeb-4d71-aa15-ce7d0c392202